examples/tensorflow/prepare.sh

```diff
@@ -0,0 +1,118 @@
+#TensorFlow ImageNet image classification prepare
+echo "Prepare tensorflow imageNet example!"
+
+function imageNet_prepare_data(){
+    #download the data
+    echo "Downloading imageNet data, waiting..."
+	chmod u+x slim/datasets/download_and_convert_imagenet.sh
+    abspath=`pwd`/data
+    echo -e "openpai\nopenpai\n" | ./slim/datasets/download_and_convert_imagenet.sh $abspath
+
+    #upload the data to HDFS
+    echo "Uploading imageNet data, waiting..."
+    for i in `ls data`
+    do
+        hdfs dfs -put data/$i hdfs://$1/examples/tensorflow/imageNet/data
+    done
+}
+
+function imageNet_prepare_code(){
+    #download the code
+	echo "Downloading imageNet code, waiting..."
+    git clone https://github.com/tensorflow/models.git && mv models/research/slim . && rm -rf models
+    
+    #edit download_and_convert_imagenet.sh
+    sed -i "s/^WORK_DIR=.*$/WORK_DIR=.\/slim/g" slim/datasets/download_and_convert_imagenet.sh
+
+    #upload the code to HDFS
+    echo "Uploading imageNet code, waiting..."
+    for i in `ls slim`
+    do
+        hdfs dfs -put slim/$i hdfs://$1/examples/tensorflow/imageNet/code
+    done
+}
+
+echo "You must input hdfs socket as the only parameter! Or you cannot run this script correctly!"
+
+#make directory on HDFS
+echo "Make imageNet directory, waiting..."
+hdfs dfs -mkdir -p hdfs://$1/examples/
+hdfs dfs -mkdir -p hdfs://$1/examples/tensorflow/
+hdfs dfs -mkdir -p hdfs://$1/examples/tensorflow/imageNet/
+hdfs dfs -mkdir -p hdfs://$1/examples/tensorflow/imageNet/data/
+hdfs dfs -mkdir -p hdfs://$1/examples/tensorflow/imageNet/code/
+hdfs dfs -mkdir -p hdfs://$1/examples/tensorflow/imageNet/output/
+
+echo "We have to ignore imageNet example now due to its big input data!"
+echo "If you want to run the imageNet preparing script, just remove the # of #imageNet_prepare_code \$1 and #imageNet_prepare_data \$1"
```

Comment:
The output here is confusing, 
If imageNet example is ignored here, we should add instruction in doc to tell user how to run them.

Reply:
done